vehicle/as/CMakeLists.txt

```diff
@@ -48,15 +48,15 @@ ament_export_dependencies(${AS_DEPENDENCIES})
 
 
 install(TARGETS ssc_interface
-  ARCHIVE DESTINATION lib/${PROJECT_NAME}
-  LIBRARY DESTINATION lib/${PROJECT_NAME}
-  RUNTIME DESTINATION lib/${PROJECT_NAME}
+  ARCHIVE DESTINATION bin/${PROJECT_NAME}
```

Comment:
Sorry, I was a bit unclear: Only the RUNTIME should go to `bin`, the other two to `lib`

Reply:
I'm sorry, and thank you for your rapid and detailed review. Very helpful!

Reply:
Now I'm going to use `ament_auto`.

Reply:
:+1: